pywbem_mock/_baseprovider.py

```diff
@@ -397,8 +460,11 @@ class are returned. `None` means not supplied by client and the
           local_only, etc. arguments.
```

Comment:
Add the return type to the Returns section, to make Sphinx recognize it properly:

```
        Returns:
          :class:`~pywbem.CIMClass`: Copy of the CIM class ...
```

Reply:
DONE